apps/site/src/app/(home)/sections/Sponsors/Sponsors.module.scss

```diff
@@ -8,61 +8,68 @@
 
 .title {
 	text-align: center;
-}
-
-.clipboard {
-	position: relative;
-	padding: 64px 32px 48px 32px;
-	min-height: 512px;
-	width: 80%;
-	margin: 0 auto;
-	border: 20px solid #aa703c;
-	border-radius: 100px;
-	background: #ffffff;
-	display: flex;
-	flex-direction: column;
-	align-items: stretch;
+	color: theme.$white;
+	font-size: 3rem;
+	font-weight: bold;
+	margin-bottom: 3rem;
 
-	@include utils.media-breakpoint-up(sm) {
-		padding: 32px 32px 48px 32px;
+	@include utils.media-breakpoint-down(sm) {
+		font-size: 2rem;
 	}
 }
 
-.clip {
-	height: 192px;
-	position: absolute;
-	left: 50%;
-	transform: translateX(-50%) rotate(90deg);
-	top: -105px;
-
-	@include utils.media-breakpoint-up(sm) {
-		height: 256px;
-		top: 50%;
-		transform: translateY(-50%);
-		left: -85px;
-	}
+.logos {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	gap: 1.5rem;
+	max-width: 1200px;
+	margin: 0 auto;
 }
 
-.logos {
-	flex-grow: 1;
-	display: grid;
+.logo {
+	width: calc(25% - 1.5rem);
+	aspect-ratio: 16 / 9;
+	background-color: theme.$white;
+	border-radius: 8px;
+	display: flex;
 	align-items: center;
-	align-content: center;
 	justify-content: center;
-	grid-template-columns: repeat(auto-fill, 192px);
-	gap: 32px;
+	transition: transform 0.3s ease;
+
+	&:hover {
+		transform: scale(1.05);
+	}
 
-	@include utils.media-breakpoint-up(sm) {
-		padding-left: 24px;
-		grid-template-columns: repeat(auto-fill, 256px);
+	@include utils.media-breakpoint-down(md) {
+		width: calc(33.333% - 1.5rem);
 	}
 
-	@include utils.media-breakpoint-up(md) {
-		grid-template-columns: repeat(auto-fill, 384px);
-		padding-left: 48px;
+	@include utils.media-breakpoint-down(sm) {
+		width: calc(50% - 1.5rem);
+	}
+
+	img {
+		max-width: 80%;
+		max-height: 80%;
+		object-fit: contain;
 	}
 }
 
-.logo {
+.placeholderLogo {
 	width: 100%;
+	height: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
```

Comment:
Suggestion: since the parent elements have `.logo` class applied, the centered placement should already be taken care of, so these shouldn't need to be specified.

Reply:
Think you also have to remove the forced dimensions (`width` and `height`) for it to be centered by parent